pages/learn/deploy/deployment.md

```diff
@@ -232,8 +232,6 @@ It should be noted that in this case the devices will still pull the container i
 
 Often it is necessary to use passwords or secrets during your build to fetch proprietary files or code but not have these sensitive files be downloaded to all the devices. For this reason `balena push` allows one to define a `.balena` folder to hold secret files which will get exposed to during the image build but not propagate down to devices.
```

Comment:
"will get exposed to during the image build" - is there a word missing between 'to' and 'during' ?